workers/unity/Packages/io.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -337,7 +338,7 @@ private static void UpdateModules()
             var gdkItemGroups = projectNode
                 .Elements("ItemGroup")
                 .Where(ele => ele.Element("GdkPackageSource") != null)
-                .ToDictionary(ele => ele.Element("GdkPackageSource").Attribute("Include").Value, ele => ele);
+                .ToDictionary(ele => ele.Element("GdkPackageSource").Attribute("Remove").Value, ele => ele);
```

Comment:
This stops a phantom folder appearing in the `csproj` if you open it in VS/Rider.